.github/workflows/build.yml.in

```diff
@@ -72,7 +72,7 @@ jobs:
   build:
     if: github.repository MAIN_OR_FORK 'leanprover-community/mathlib4'
     name: BuildJOB_NAME
-    runs-on: ubuntu-latest
+    runs-on: pr
```

Comment:
This is a variable (`RUNS_ON`) in mathlib, to avoid using the hoskinson machines in forks (and to use the `bors` machine for staging commits, I guess). Is it worth doing the same here?

Reply:
Thanks, I've changed to using this mechanism.